compliance_suite/report_server.py

```diff
@@ -7,6 +7,7 @@
 import http.server
 import json
 import os
+from typing import List
```

Comment:
Please follow import ordering used in library. Not sure how @LakiG deals with `from x import y` versus `import x`, but it seems to me that at least within each of these, there is alphabetical sorting according to the package name.

Reply:
Yeah, there doesnt seem to be any guideline about this. Ref - https://peps.python.org/pep-0008/#imports
But within the same import group, it would be better to separate them. Example:

`import x` _All alphabetically sorted_
`from x import y` _All alphabetically sorted_